execution_workflows/APAtrap/README.md

```diff
@@ -0,0 +1,56 @@
+# APAeval nextflow pilot run
+ 
+ # APAtrap
+The APAtrap R package does the following: 
+1. Refines annotated 3'UTR and identifies novel 3' UTRs and 3' UTR extensions
+2. Aims to identify all potential APA (alternative polyadenylation) sites
+3. Detects genes with differential APA site usage between conditions by leveraging 
+   the resolution of RNA-seq data
+
+The [paper](https://academic.oup.com/bioinformatics/article/34/11/1841/4816794) is titled APAtrap: identification and quantification of 
+alternative polyadenylation sites from RNA-seq data
+The [application download](https://sourceforge.net/projects/apatrap/files/) is free to download
+and [user manual](https://sourceforge.net/p/apatrap/wiki/User%20Manual/) was used as a referrence
+to create the nextflow pipeline flow of this module
+
+### Steps to run this:
+ - Replace "path_to" in samplesheet_example_files.csv with the path to the reference input files
+ - Check the path to the input files with `pwd` and replace the `path_to` in samplesheet_example_files.csv with the path from the `pwd` command
+ - Pull the required docker image by doing `docker pull docker.io/faricazjj/apatrap:latest`
```

Comment:
update docker link to `docker.io/apaeval/apatrap:latest` as in rest of pipeline

Reply:
deleting this line since it's not necessary to pull the required docker image--the image is automatically pulled when nextflow runs